lib/rules/web/admin_console/4.6/clustersettings.xyaml

```diff
@@ -0,0 +1,32 @@
+check_ocm_subscription:
+  element:
+    selector:
+      xpath: //*[contains(., 'OpenShift Cluster Manager') and @href='https://cloud.redhat.com/openshift/details/<cluster_id>']
```

Comment:
We can use `//a` instead of `//*` since for an external link, the tag will always be `//a`